.github/workflows/vite-gh-pages.yml

```
@@ -0,0 +1,75 @@
+  cancel-in-progress: false
+
+jobs:
+  # Build job
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup Pages
+        uses: actions/configure-pages@v5
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)